finite-state/shared/src/main/scala/fs2/data/esp/Guard.scala

```diff
@@ -1,104 +0,0 @@
-/*
```

Comment:
Was it not worth to keep this a possible implementation for guards even though it's no longer the exclusive one? Seemed general enough to be useful several times.

Reply:
Actually the `In` and `NotIn` operators and their evaluation was terribly specific and redundant with the XPath predicate that we have.

Reply:
OK then.